src/scripts/reducers/VideosReducer.js

```diff
@@ -27,7 +27,7 @@ import type { Action, VideoRecordMap } from 'types/ApplicationTypes'
 const reducer = {
   [UPLOAD_REQUESTED]: (
     state: VideoRecordMap,
-    { payload }: Action<{ id: string, filename: string }> = {}
```

Comment:
I would keep the `= {}` default just in case we call the action w/ no payload (to avoid runtime error)